chef/cookbooks/crowbar/recipes/prepare-upgrade-scripts.rb

```diff
@@ -175,6 +175,14 @@
   action :create
 end
 
+template "/usr/sbin/crowbar-set-network-agents-state.sh" do
+  source "crowbar-set-network-agents-state.sh.erb"
+  mode "0755"
+  owner "root"
+  group "root"
+  action :create
+end
```

Comment:
Do we need to place the script on all nodes? (Not that it really hurts)

Reply:
Not really. But the same is true for the `crowbar-router-migration.sh` script above. Both are only needed on the "controller" nodes. (Not neutron-network)